waku/v2/protocol/waku_rln_relay/waku_rln_relay_utils.nim

```diff
@@ -422,23 +445,24 @@ proc getCurrentEpoch*(): Epoch =
 
 proc compare*(e1, e2: Epoch): int64 =
```

Comment:
To my intuition "compare" would return -1,0,1 depending if e1 is lower, equal, greater than e2 (as happens for the `cmp` function common to many languages). Definitely not relevant, but maybe clearer to call it diff, epochGap or something like this?

Reply:
Makes sense, it is now renamed to `diff`.  https://github.com/status-im/nim-waku/pull/941/commits/fbd5d53b55aafc2e2b936792b415c8f2d4ae83e0